x-pack/plugins/observability_solution/apm/server/routes/services/get_service_instance_metadata_details.ts

```diff
@@ -85,11 +87,12 @@ export async function getServiceInstanceMetadataDetails({
           terminate_after: 1,
           size: 1,
           query: { bool: { filter } },
+          fields: ['*'],
         },
       }
     );
 
-    return maybe(response.hits.hits[0]?._source);
+    return maybe(serviceInstanceMetadataDetailsMapping(response.hits.hits[0]?.fields));
```

Comment:
is that the right mapping function here? looks wrong

Reply:
From what I saw we have the same fields in the response so I assume what @miloszmarcinkowski  created as a mapper would cover the other queries as well.
As this query was using the whole `_source` before and wasn't mapped here it should be enough to use the mapper - @dgieselaar can you please take a look when you have time if the mapper looks correct to you as well?